package.json

```diff
@@ -25,5 +26,9 @@
   "packageManager": "pnpm@8.15.6",
   "engines": {
     "node": ">=20"
+  },
+  "dependencies": {
+    "drizzle-orm": "0.31.4",
+    "pg": "8.12.0"
```

Comment:
Dependencies that are shared by `packages/core` and `apps/web` but start different instances gives these error
```
Argument of type 'NodePgDatabase<typeof import("/Users/andres/code/latitude/llm/packages/core/src/schema/index")>' is not assignable to parameter of type 'PgDatabase<any, any, any>'.
  The types of '_.session' are incompatible between these types.
    Type 'PgSession<NodePgQueryResultHKT, typeof import("/Users/andres/code/latitude/llm/packages/core/src/schema/index"), ExtractTablesWithRelations<typeof import("/Users/andres/code/latitude/llm/packages/core/src/schema/index")>>' is not assignable to type 'PgSession<any, any, any>'.
      Property 'dialect' is protected but type 'PgSession<TQueryResult, TFullSchema, TSchema>' is not a class derived from 'PgSession<TQueryResult, TFullSchema, TSchema>'.

```

![image](https://github.com/user-attachments/assets/cf838fcd-5920-46fc-9849-f371e8b8ee05)

Having it in the root `package.json` remove these errors